data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -1,91 +1,34 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.0
```

Comment:
After I opened the notebook from the jupyter UI it automatically changes the jupytext version to 1.7.1. I think it should be the same version as what is in `requirements.txt`.
```suggestion
#       jupytext_version: 1.7.1
```
Note: Currently this yaml front-matter is required to be in each notebook. In the future (not now) I would like to find a way to configure this in just one place. That way if we e.g. bump jupytext versions, it would only have to be changed in one place.